robot_calibration/include/robot_calibration/ceres/camera3d_to_arm_error.h

```diff
@@ -106,11 +106,27 @@ struct Camera3dToArmError
                                      CalibrationOffsetParser* offsets,
                                      robot_calibration_msgs::CalibrationData& data)
   {
+    int index = -1;
+    for (size_t k = 0; k < data.observations.size() ; k++)
+    {
+      if ( data.observations[k].sensor_name == "camera")
```

Comment:
"camera" should probably not be hard coded -- but it looks like we don't currently expose the camera model name -- so we probably need to adjust that API